### PR TITLE
feat: Added action apply-state

### PR DESCRIPTION
**Pull request by diegolagospagopa**
This action is needed by azuredevops pipelines, when they want to apply an output state generated before

### CODE REVIEW

File: azure/terraform.sh [hunk 1]
```diff
@@ -149,6 +159,8 @@ env=$2
 shift 2
 other=$@
 
+subscription="NOT_LOADED"
```
Comment (mamari90) [inline azure/terraform.sh:162]: why is this needed?

File: azure/terraform.sh [hunk 2]
```diff
@@ -75,6 +75,16 @@ function other_actions() {
   fi
 }
 
+## Apply state don't have -var-file or vars because are inherith from output state
+function apply_state_action() {
```
Comment (fmlisco) [inline azure/terraform.sh:79]: 1) tbh, I didn't get what this function does and what an "output state" is.
2) Furthermore, there is no need to create another function. You should use the already present "other_actions" function just adding another case to it (via "if" or "case" statement)
3) Lastly, you should increment the "VERSION" variable at the beginning of this script, otherwise it's not possible to know what script is the last version of it

**Review by fmlisco — COMMENTED** [review]
please check the comments

File: azure/terraform.sh [hunk 3]
```diff
@@ -75,6 +75,16 @@ function other_actions() {
   fi
 }
 
+## Apply state don't have -var-file or vars because are inherith from output state
```
Comment (fmlisco) [inline azure/terraform.sh:78]: "The apply state does not require a -var-file or vars because these are inherited from the output state."?

File: azure/terraform.sh (hunk 1 above)
Comment (diegolagospagopa) [reply azure/terraform.sh:162]: shellcheck tip

File: azure/terraform.sh (hunk 2 above)
Comment (diegolagospagopa) [reply azure/terraform.sh:79]: ok i will change the others function

File: azure/terraform.sh (hunk 3 above)
Comment (diegolagospagopa) [reply azure/terraform.sh:78]: done

**Review by mamari90 — APPROVED** [review]
